CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ project(
   atsdk
   VERSION 0.0.1
```

Comment:
This should be tracking the version we use for releases

Reply:
Okay, I will make sure to update it in the next release bump